tests/travis/setup.sh

```diff
@@ -15,21 +15,3 @@ if [ ${TASK} == "lint" ]; then
     conda info -a
     conda create -n python3 python=3.7
 fi
-
-
-if [ ${TASK} == "python_test" ] || [ ${TASK} == "python_lightweight_test" ] || [ ${TASK} ==  "distributed_test" ]; then
```

Comment:
Perhaps I am missing something here but are we removing lint + python_lightweight_test?

Reply:
They are being migrated to Jenkins. (See lint in Formatting Check stage.) I didn’t add flake8 yet, will follow up with another PR.

Reply:
Sounds good. Thanks.